src/main/java/com/ekart/model/Product.java

```diff
@@ -108,4 +108,20 @@
     @Type(type = "org.hibernate.type.ImageType")
     @JsonIgnore
     private List<byte[]> images;
+
+    public Product(String name, Category category, Brand brand, Set<Size> size, Color color, List<String> description,
```

Comment:
Don't we have any alternative in Lombok for this?

Reply:
We aren't constructing full entity. Since it's an partial entity there is no alternative. Alternative would be to use setters

Reply:
ok